internal/jasmine_node_test/jasmine_runner.js

```diff
@@ -63,6 +63,9 @@ function main(args) {
   process.argv.splice(2, 1)[0];
```

Comment:
we take care here to pass the residual argv as-is to jasmine. Will adding a new config file argument potentially confuse any test code that reads the argv? I wish the comment here gave more info about scenarios where the tested code processes the argv...

Reply:
This is a valid concern and request.

I wish there were clearer handling of command line args in code instead of in comment though while working on this chunk of code.